internal/acceptance/lakeview_dashboard_test.go

```diff
@@ -0,0 +1,489 @@
+			require.NoError(t, err)
+			// As the format of the serialized dashboard is not fixed, we can only check if it is not empty
+			assert.NotEqual(t, "", dashboard.SerializedDashboard)
+			require.NotEqual(t, dashboard.UpdateTime, dashboard.CreateTime)
+			resource.TestCheckResourceAttr("databricks_lakeview_dashboard.d1", "etag", dashboard.Etag)
+			return nil
+		}),
+	})
+}
+
+func TestAccDashboardTestAll(t *testing.T) {
+	dashboard_id := ""
+	display_name := ""
+	warehouse_id := ""
+	etag := ""
+	tmpDir := fmt.Sprintf("/tmp/Lakeview_dashboard-%s", qa.RandomName())
+	fileName := tmpDir + "/Dashboard.json"
+	var template templateStruct
+	workspaceLevel(t, step{
+		PreConfig: func() {
+			os.Mkdir(tmpDir, 0755)
+			os.WriteFile(fileName, []byte("{\"pages\":[{\"name\":\"new_name\",\"displayName\":\"New Page in file\"}]}"), 0644)
+
+		},
+		Template: makeTemplate(template.SetAttributes(map[string]string{
+			"display_name":      "Monthly Traffic Report",
+			"warehouse_id":      "{env.TEST_DEFAULT_WAREHOUSE_ID}",
+			"parent_path":       "/Shared/provider-test",
+			"file_path":         fileName,
+			"embed_credentials": "false",
+		})),
+		Check: resourceCheck("databricks_lakeview_dashboard.d1", func(ctx context.Context, client *common.DatabricksClient, id string) error {
+			w, err := client.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			dashboard, err := w.Lakeview.Get(ctx, dashboards.GetDashboardRequest{
+				DashboardId: id,
+			})
+			if err != nil {
+				return err
+			}
+			publish_dash, err := w.Lakeview.GetPublished(ctx, dashboards.GetPublishedDashboardRequest{
+				DashboardId: id,
+			})
+			assert.Equal(t, "Monthly Traffic Report", dashboard.DisplayName)
+			require.NoError(t, err)
+			require.NotEqual(t, dashboard.UpdateTime, dashboard.CreateTime)
+			require.Equal(t, publish_dash.EmbedCredentials, false)
+			return nil
+		}),
+	}, step{
+		PreConfig: func() {
+			os.WriteFile(fileName, []byte("{\"pages\":[{\"name\":\"new_name\",\"displayName\":\"New Page Modified\"}]}"), 0644)
+		},
+		Template: makeTemplate(template),
+		Check: resourceCheck("databricks_lakeview_dashboard.d1", func(ctx context.Context, client *common.DatabricksClient, id string) error {
+			w, err := client.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			dashboard, err := w.Lakeview.Get(ctx, dashboards.GetDashboardRequest{
+				DashboardId: id,
+			})
+			if err != nil {
+				return err
+			}
+			assert.Equal(t, "Monthly Traffic Report", dashboard.DisplayName)
+			require.NoError(t, err)
+			dashboard_id = dashboard.DashboardId
+			display_name = dashboard.DisplayName
+			warehouse_id = dashboard.WarehouseId
+			etag = dashboard.Etag
+			// As the format of the serialized dashboard is not fixed, we can only check if it is not empty
+			assert.NotEqual(t, "", dashboard.SerializedDashboard)
+			return nil
+		}),
+	}, step{
+		PreConfig: func() {
+			w, err := databricks.NewWorkspaceClient(&databricks.Config{})
+			if err != nil {
```

Comment:
Test should fail if there is an error. Use this instead of logging (it will also log automatically)
```
require.NoError(t, err)
```